app/models/tv_series.rb

```diff
@@ -29,7 +46,19 @@ def self.parse_results(result, show_id)
     @overview = result[:overview]
     @seasons = (1..(result[:number_of_seasons])).to_a if result[:number_of_seasons].present?
     @actors = TVCast.parse_results(result[:credits][:cast])
-    @series = TVSeries.new(@show_id, @first_air_date, @last_air_date, @show_name, @backdrop_path, @poster_path, @number_of_episodes, @number_of_seasons, @overview, @seasons, @actors)
-  end #parse results
 
-end #class
\ No newline at end of file
+    @series = TVSeries.new(
```

Comment:
I needed to change this line because of the keyword argument refactor above. I'd have preferred to roll lines 38-48 above in to these lines below, however, I am not sure where all of the instance variables are being called further down the line. I can do a separate cleanup PR to address this. In the meantime, I chose to keep the scope of this PR as small as possible.

Reply:
🎉